React/Views/ScrollView/RCTScrollView.m

```diff
@@ -678,9 +678,21 @@ - (void)dealloc
   [_eventDispatcher.bridge.uiManager.observerCoordinator removeObserver:self];
 }
 
+#if TARGET_OS_OSX // TODO(macOS ISS#2323203)
+- (void)layout
+{
+  [super layout];
+  [self commonLayoutImplementation];
+}
+#else
 - (void)layoutSubviews
 {
   [super layoutSubviews];
+  [self commonLayoutImplementation];
+}
+#endif // TODO(macOS ISS#2323203)
+
+- (void)commonLayoutImplementation {
```

Comment:
I know it's only me to blame, but I don't like this method name. Still can't think of anything better that captures  what's actually happening here.